doc/md/guides/orms/efcore.md

```diff
@@ -0,0 +1,276 @@
+---
+id: efcore
+title: Automatic Migration Planning for Entity Framework Core
+slug: /guides/orms/efcore
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## TL;DR
+* [Entity Framework (EF) Core](https://docs.microsoft.com/en-us/ef/core/) is an object-relational mapping (ORM) framework for .NET.
+* [Atlas](https://atlasgo.io) is an open-source tool for inspecting, planning, linting, and executing schema changes to your database.
+* Developers using EF Core can use Atlas to automatically plan schema migrations
+ based on the desired state of their schema instead of crafting them by hand.
+
+## Automatic Migration Planning for EF Core
+
+EF Core is one of the most popular ORMs widely used in the .NET community, supported by **Microsoft**. EF Core allows 
+users to manage their database schemas using its [migrations](https://docs.microsoft.com/en-us/ef/core/managing-schemas/migrations/) 
+feature, which is usually sufficient during development and in many simple cases.
+
+However, there are two well-known issues with EF Core migrations ([efcore#31790](https://github.com/dotnet/efcore/issues/31790)):
+
+1. **The SQL is not transparently visible in merge/pull requests for review purposes**: 
+Migration files generated by EF Core do follow the _versioned migrations_ pattern, but each 
+file contains the operations necessary to apply the migration (in `Up`) and to revert it (in `Down`), instead 
+of the SQL statements that would be executed in a versioned migration.
+
+```csharp
+public partial class Initial : Migration
+{
+  /// <inheritdoc />
+  protected override void Up(MigrationBuilder migrationBuilder)
+  {
+    migrationBuilder.CreateTable(
+      name: "AspNetRoles",
+      columns: table => new
+      {
+        Id = table.Column<string>(type: "TEXT", nullable: false),
+        Name = table.Column<string>(type: "TEXT", maxLength: 256, nullable: true),
+        NormalizedName = table.Column<string>(type: "TEXT", maxLength: 256, nullable: true),
+        ConcurrencyStamp1 = table.Column<string>(type: "TEXT", nullable: true)
+      },
+      constraints: table =>
+      {
+        table.PrimaryKey("PK_AspNetRoles", x => x.Id);
+      });
+    ...
+  }
+}
+
+```
+
+2. **The designer files repeat the entire schema for each migration**: Along with migration files, EF Core also
```

Comment:
as a  non dotnet user, i have no idea what they are. define in one sentence and provide a link to learn more